ops/webapp.conf

```diff
@@ -1,4 +1,7 @@
+passenger_log_file /dev/stdout;
```

Comment:
did you remove or disable the log tailer? passenger images currently runs a process that tees the logs in to stdout, which we should disable if we are going about this directly instead.